pandas/tests/frame/test_apply.py

```diff
@@ -1392,3 +1392,101 @@ def test_consistency_of_aggregates_of_columns_with_missing_values(self, df, meth
         tm.assert_series_equal(
             none_in_first_column_result, none_in_second_column_result
         )
+
+
+class TestDataFrameNamedAggregate:
```

Comment:
can you move these to a new file test_aggregate.py

Reply:
and any other agg tests in test_apply as well

Reply:
you can move existing tests in a followup (or pre-cursor) PR if easier / more clear

Reply:
yeah, I would prefer to have a followup PR for moving around, is it okay? the main reason is I added a helper function in `aggregate.py` in this PR, and in case if any changes needed, i would prefer to change it in this PR since it is closely related to `series.agg` and `frame.agg`